runtime/src/common/registry.rs

```diff
@@ -0,0 +1,35 @@
+//! Registry structures.
+//!
+//! # Note
+//!
+//! This **MUST** be kept in sync with go/registry/api.
+//!
+use super::{
+    super::storage::mkvs::WriteLog,
+    crypto::{hash, signature::SignatureBundle},
+};
+use serde_derive::{Deserialize, Serialize};
+
+// RuntimeGenesis is the runtime genesis information that is used to
```

Comment:
Should use Rust documentation comments (`///`) and not include the `<name> is` prefix. Also for fields below.